modules/administration-guide/pages/configuring-che-storage-strategy.adoc

```diff
@@ -0,0 +1,34 @@
+:_content-type: CONCEPT
+:description: Configuring the storage strategy in {prod-short} 
+:keywords: administration guide, configuring, {prod-short}, storage, strategy
+:navtitle: Configuring the storage strategy in {prod-short}
+:page-aliases: installation-guide:configuring-che-storage-strategy.adoc
+
+[id="configuring-{prod-id-short}-storage-strategy"]
+= Configuring the storage strategy in {prod-short}
+
+{prod-short} can be configured to provide persistent or non-persistent storage to workspaces by selecting a storage strategy. The selected storage strategy will be applied to all newly created workspaces by default. Users can opt for a non-default storage strategy for their workspace in their xref:end-user-guide:requesting-persistent-storage-for-workspaces.adoc[devfile] or through the xref:end-user-guide:url-parameter-for-the-workspace-storage.adoc[URL parameter].
+
+.Available storage strategies:
+
+* `per-user`: Use a single PVC for all workspaces created by a user.
+* `per-workspace`: Each workspace is given its own PVC.
+* `ephemeral`: Non-persistent storage; any local changes will be lost when the workspace is stopped.
+
+The default storage strategy used in {prod-short} is `per-user`. 
+
+.Procedure
```

Comment:
Same comment as in https://github.com/eclipse-che/che-docs/pull/2536/files#r1157451521.